docs/client-concepts/high-level/indexing/ingest-nodes.asciidoc

```diff
@@ -0,0 +1,42 @@
+:ref_current: https://www.elastic.co/guide/en/elasticsearch/reference/6.x
+
+:github: https://github.com/elastic/elasticsearch-net
+
+:nuget: https://www.nuget.org/packages
+
+////
+IMPORTANT NOTE
+==============
+This file has been generated from https://github.com/elastic/elasticsearch-net/tree/6.x/src/Tests/Tests/ClientConcepts/HighLevel/Indexing/IngestNodes.doc.cs. 
+If you wish to submit a PR for any spelling mistakes, typos or grammatical errors for this file,
+please modify the original csharp file found at the link and submit the PR with that change. Thanks!
+////
+
+[[ingest-nodes]]
+=== Ingest Nodes
```

Comment:
Would recommend Ingest Node as the title, which [aligns with Elasticsearch reference](https://www.elastic.co/guide/en/elasticsearch/reference/6.6/ingest.html)

Reply:
Good idea for SEO and for aligning with the ref docs.